### PR TITLE
DE46767 - Quizzing completion summary for unpublished quizzes

### PR DESCRIPTION
**Pull request by maddiegabriel**
[DE46767 - [Consistent Evaluation][Quizzing] Fix completion summary for unpublished quizzes](https://rally1.rallydev.com/#/42960320374d/custom/236803223728?detail=%2Fdefect%2F622411218729
)

Previously, the Grade Statistics section was not rendering at all for students with no attempt evaluations published.

Now we render the header with a message for the user:

This is different from legacy, where we rendered empty boxes (ran by design):

### CODE REVIEW

File: components/left-panel/quizzing/consistent-evaluation-quizzing-overall-summary.js [hunk 1]
```diff
@@ -174,43 +193,16 @@ export class ConsistentEvaluationQuizzingOverallSummary extends SkeletonMixin(Rt
 		return html`${this.localize(isTimeLimitEnforced, { timeLimit: formatSeconds(this.quizSummaryInfo.timeLimit ? this.quizSummaryInfo.timeLimit : 0) })}`;
 	}
 
-	_renderGradeStatisticsSkeleton() {
```
Comment (maddiegabriel) [inline components/left-panel/quizzing/consistent-evaluation-quizzing-overall-summary.js:177]: Able to remove skeleton scaffolding with these changes

File: components/left-panel/quizzing/consistent-evaluation-quizzing-overall-summary.js [hunk 2]
```diff
@@ -60,14 +60,13 @@ export class ConsistentEvaluationQuizzingOverallSummary extends SkeletonMixin(Rt
...
-				padding-left: 1rem;
-				padding-right: 1rem;
+			.d2l-consistent-eval-grade-stats-container {
+				margin: 0.5rem 1rem;
 			}
 			.d2l-list-item-stats div {
 				font-weight: bold;
+				min-height: 1rem;
 				padding-left: 0.625rem;
-				padding-right: 0.625rem;
```
Comment (Scy-D2L) [inline components/left-panel/quizzing/consistent-evaluation-quizzing-overall-summary.js:70]: Has this been tested with RTL?

**Review by Scy-D2L — APPROVED** [review]
Lgtm! Thanks for tidying up around the skeletizing.

File: components/left-panel/quizzing/consistent-evaluation-quizzing-overall-summary.js (hunk 2 above)
Comment (maddiegabriel) [reply components/left-panel/quizzing/consistent-evaluation-quizzing-overall-summary.js:70]: now it has 😆 good catch

**Review by JamieTran — APPROVED** [review]
lgtm!

**Pull request merged**